R/Previewer.R

```diff
@@ -105,18 +108,32 @@ reporter_previewer_srv <- function(id,
           showrcode = any_rcode_block(reporter),
           session = session
         ),
-        htmltools::tagAppendAttributes(
-          shiny::tags$a(
-            id = ns("download_data_prev"),
-            class = "btn btn-primary shiny-download-link",
-            href = "",
-            target = "_blank",
-            download = NA,
-            shiny::tags$span("Download Report", shiny::icon("download"))
+        shiny::tags$div(
+          id = "previewer_reporter_buttons",
+          class = "previewer_buttons_line",
+          htmltools::tagAppendAttributes(
+            shiny::tags$a(
+              id = ns("download_data_prev"),
+              class = "btn btn-primary shiny-download-link simple_report_button",
+              href = "",
+              target = "_blank",
+              download = NA,
+              shiny::tags$span("Download Report", shiny::icon("download"))
+            ),
+            class = if (nr_cards) "" else "disabled"
           ),
-          class = if (length(reporter$get_cards())) "" else "disabled"
-        ),
-        reset_report_button_ui(ns("resetButtonPreviewer"), label = "Reset Report")
+          shiny::tags$button(
```

Comment:
Can you please add `reporter_previewer_srv(reporter_buttons = c("add", "download", "load", "reset"))` argument and render only those specified. In the same time please update `teal::reporter_previewe_module` and pass `c("add", "download", "reset")` to the [`reporter_previewer_module(server_args`](https://github.com/insightsengineering/teal/blob/a358d6398bb9677a26da47060e4e7214311cce80/R/reporter_previewer_module.R#L22)?